go.mod

```diff
@@ -171,10 +171,10 @@ require (
 
 replace (
 	// Knative forks.
-	knative.dev/eventing => github.com/openshift-knative/eventing v0.99.1-0.20221215170220-da8c25a653e7
+	knative.dev/eventing => github.com/openshift-knative/eventing v0.99.1-0.20221223064251-7859459225d6
 	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20221109100552-26eea5293bca
```

Comment:
In a separate PR I'd also update the `eventing-kafka` b/c of https://github.com/openshift-knative/eventing-kafka/commit/81b8c0ceb5408ff1d75f016fdd8e9f455b17ad4c

Reply:
Yeah lets just update the manifests, then all the go deps in one go (as there are conflicts between eventing & knative-operator, if done separately).